pkg/operator/ceph/object/spec.go

```diff
@@ -34,7 +35,22 @@ import (
 )
 
 const (
-	livenessProbePath = "/swift/healthcheck"
+	livenessProbePath   = "/swift/healthcheck"
+	authProvider        = "KMS_PROVIDER"
+	vaultAddr           = "VAULT_ADDR"
+	vaultBackendPath    = "VAULT_BACKEND_PATH"
+	setupVaultTokenFile = `
+set -xe
+
+VAULT_TOKEN_OLD_PATH=%s
+VAULT_TOKEN_NEW_PATH=%s
+
+cp $VAULT_TOKEN_OLD_PATH $VAULT_TOKEN_NEW_PATH
```

Comment:
Why do we need this init container? Why `cp`?

Reply:
The vault token is passed as Secret for rgw container. So it is mounted as read-only. RGW has restrictions over vault token file, it should be owned by the same user(ceph in our case) which rgw daemon runs and all other permission should be nil or zero. Here ownership can be changed with help of `FSGroup` but in openshift environment, it won't work. Hence the token file is copied to `containerDataDir` from mounted secret then ownership/permissions are changed accordingly with help of a init container.

Reply:
Looks like this init container fails in the CI: `encryption-kms-get-kek`

Reply:
Let's add `--verbose` to  `cp`.